src/app-layout/visual-refresh/context.tsx

```diff
@@ -36,7 +36,7 @@ import { useUniqueId } from '../../internal/hooks/use-unique-id';
 
 interface AppLayoutInternals extends AppLayoutProps {
   activeDrawerId: string | undefined;
-  drawers: Array<DrawerItem>;
+  drawers: Array<DrawerItem> | null;
```

Comment:
Empty array `[]` means that `<AppLayout drawers={[]}>` is used and `null` means that there are no drawers registered in any form. This distinction is needed to decide what we render on the right hand side